ddtrace/internal/processor/trace.py

```diff
@@ -141,6 +141,9 @@ def on_span_finish(self, span):
                 else:
                     finished = trace_spans
 
+                if self._partial_flush_enabled and trace.num_finished >= self._partial_flush_min_spans:
+                    log.debug("Partially flushing %d spans for trace %d", len(trace.spans), span.trace_id)
```

Comment:
```suggestion
                    log.debug("Partially flushing %d spans for trace %d", len(finished), span.trace_id)
```

Reply:
we should also add a test case for this log line as well

Reply:
I started writing a test for `Partially flushing %d spans for trace` in the test_integration.py file, but the mock call seems to return an empty log. I'm not sure why though.